docs/DataProviders.md

```diff
@@ -72,7 +72,10 @@ Due to the breaking changes, the following providers are no longer working with
 
 * **[DynamoDb](https://github.com/abiglobalhealth/aor-dynamodb-client)**: [abiglobalhealth/aor-dynamodb-client](https://github.com/abiglobalhealth/aor-dynamodb-client)
 * **[Epilogue](https://github.com/dchester/epilogue)**: [dunghuynh/aor-epilogue-client](https://github.com/dunghuynh/aor-epilogue-client)
-* **[Loopback](http://loopback.io/)**: [kimkha/aor-loopback](https://github.com/kimkha/aor-loopback)
```

Comment:
Why did you remove this one ?

Reply:
Sorry that was a mistake, I was trying to fix the first commit to allow for an automatic merge